examples/crd/crd-v1.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: tfjobs.kubeflow.org
+spec:
+  group: kubeflow.org
```

Comment:
Do we need to have  a single CRD serving two versions? (similar to install package?) WDYT?

Reply:
Done